rwengine/src/data/Weather.hpp

```diff
@@ -39,6 +39,13 @@ class Weather {
                       float a, float tod);
 
     std::vector<Entry> entries;
+
+    // TODO: This weather list applies only for GTA III
+    const uint16_t WeatherList[64] = {
+            0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 1, 1, 2, 2, 1, 0,
+            0, 0, 1, 3, 3, 1, 0, 0, 1, 1, 0, 0, 0, 0, 0, 0, 0, 0, 1, 1, 2,
+            2, 1, 2, 1, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 3, 3, 0, 0, 0, 2, 1
+    };
```

Comment:
Please document wether this comes from the game files or wether it's just typed by hand by observing what the game does.

Reply:
Documented 13c4e76